src/AchievementAction.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace RA;
+
+abstract class AchievementAction
```

Comment:
I think this should be `AchievementUpdateAction` or `Update` should be added to the individual actions. `AchievementAction::DisplayOrder` just doesn't convey the fact that DisplayOrder is being updated.

Reply:
Fair point - this was one of the reasons for marking this PR as a draft eventually. The constants should include the verb similar to the other enums, i agree.
Yet, while working on this I found it odd to even specify the kind of update as an enum - these actions may as well be separate request files doing just one kind of update. That was the goal when moving request files in the first place. I'll probably refactor all of those update requests to get rid of the "action" constants where possible and make their usage in scripts obsolete by calling dedicated endpoints explicitly.